src/ldp/AuthenticatedLdpHandler.ts

```diff
@@ -113,7 +114,19 @@ export class AuthenticatedLdpHandler extends HttpHandler {
     const op: Operation = await this.requestParser.handleSafe(request);
     const credentials: Credentials = await this.credentialsExtractor.handleSafe(request);
     const permissions: PermissionSet = await this.permissionsExtractor.handleSafe(op);
+
+    // Monitor the data stream while authorizer is checking the permissions
+    let monitor: StreamMonitor | undefined;
+    if (op.body) {
+      monitor = new StreamMonitor(op.body.data, 'AuthenticatedLdpHandler');
```

Comment:
Can we do
```suggestion
      monitor = new StreamMonitor(op.body.data, this);
```
?

Or if just for tracing purposes, we could also just rely on `new Error()` in `StreamMonitor`.